### ISSUE TITLE
Can't Get Value for Volume Fader

### ISSUE THREAD

**Issue by chigkim** [chigkim/Flogic#3]

**Comment by chigkim** [comment]
No rush, but just curious... Any progress on this?
FYI, select all and delete the original message and include only reply when you respond to Github generated emails.
It looks cleaner on the web and hides email addresses.

**Comment by chigkim** [comment]
Instead of silence, I think I'll probably make it say volume up or 
volume down.

**Comment by bryansmart** [comment]
Setting volume levels is a little distracting with speech chattering, but I understand how some sort of feedback is desirable.

Are you doing much with the verbosity setting yet? Maybe only speak the message when FLogic is verbose.

**Comment by chigkim** [comment]
Yes, For example, if you set verbosity high, it'll say mute on, if it's 
set to low, it'll just say on.
